### ISSUE TITLE
Breaking changes on v0.25.0

### ISSUE THREAD

**Issue by faustinoaq** [TechMagister/liquid.cr#33]
Hi @docelic @TechMagister 

Liquid.cr need some updates to fix breaking-changes on crystal v0.25.0

```
in lib/liquid/src/liquid/filters/first.cr:9: no overload matches 'JSON::Any.new' with type JSON::Any
Overloads are:
 - JSON::Any.new(raw : Type)
 - JSON::Any.new(pull : JSON::PullParser)
 - JSON::Any.new(raw : Int32)
 - JSON::Any.new(raw : Float32)
 - JSON::Any.new(raw : Time)
        Any.new d.first
```

^^ We're getting this on amber specs

Ref: https://github.com/amberframework/amber/pull/826

**Comment by bcardiff** [comment]
@faustinoaq https://github.com/TechMagister/liquid.cr/pull/32 ;-)

**Comment by docelic** [comment]
Solved by merging #32, thanks @bcardiff and @faustinoaq :)
